src/connector/src/source/cdc/external/postgres.rs

```diff
@@ -77,6 +79,8 @@ pub struct PostgresExternalTableReader {
     config: ExternalTableConfig,
     rw_schema: Schema,
     field_names: String,
+    // store the name of uuid fields
+    pk_uuid_field_bitmap: Bitmap,
```

Comment:
Maybe too dedicated to `uuid`. What we are really missing here is a way to convert each column back to upstream's column type (`Vec<postgres_types::Type>`?). Notably we may need to map from `varchar`/`rw_int256` back into `decimal` very soon #16346 (Practically not urgent if that `decimal` is not part of pk - if it is, our null-filling logic in `rw_int256` case would also cause a problem.)

Reply:
I thought it would be better to have a `Vec<postgres_types::Type>` for the primary key, but `postgres_types::Type` doesn't have a `from_str` or `parse_str` function to parse a data type name in `String` to `Type`. So we have to do it by ourselves, which I think it is tedious right now.

Reply:
It is unnecessary to query `information_schema` first and then parse into `Type`. We can **prepare** the final query to know what `Type` it expects to see for each parameter:
https://docs.rs/tokio-postgres/latest/tokio_postgres/struct.Statement.html